cycle-scripts-es-browserify/scripts/init.js

```diff
@@ -131,12 +148,33 @@ function successMsg (appName, appPath) {
   console.log('Happy cycling!')
 }
 
-module.exports = function (appPath, appName, streamLib, verbose, originalDirectory) {
+module.exports = function (appPath, appName, libs, verbose, originalDirectory) {
   var ownPackageName = require(path.join(__dirname, '..', 'package.json')).name
   var ownPath = path.join(appPath, 'node_modules', ownPackageName)
   var appPackageJson = path.join(appPath, 'package.json')
   var appPackage = require(appPackageJson)
 
+  if (libs.test === 'ava') {
+    appPackage.ava = {
```

Comment:
There is another way to pass this parameters to the test runner? If possible, we should avoid polluting package.json with configuration that could be kept inside the `cycle-scripts-*` and put them only after eject.